monitoring/v3/src/main/java/CreateCustomMetric.java

```diff
@@ -0,0 +1,324 @@
+/**
+ * Copyright (c) 2015 Google Inc.
+ * <p/>
```

Comment:
You can make this a regular comment block instead of javadoc and remove the <p> tags.